ProjectScript_4.sql

```diff
@@ -0,0 +1,36 @@
+DELIMITER $$
+DROP PROCEDURE IF EXISTS my_project_table $$
+CREATE PROCEDURE my_project_table()
+BEGIN
+  START TRANSACTION;
+
+	# Creating a Project Table
+	CREATE TABLE Project (
+	ProjectID INT PRIMARY KEY AUTO_INCREMENT,
+	ProjectName VARCHAR(70),
+    `Value` DECIMAL (10,2),
+	Technology VARCHAR (255));
+    
+    # Inserting values into the project table
+	INSERT INTO Project (ProjectName, `Value`, Technology) VALUES
+	("Project1", 100.27, "SQL, Java, Python"), ("Project2", 1000, "SQL, Java, RubyOnRails"),
```

Comment:
Some indentation issues here

Reply:
I would also probably rather see each row on a new line. I thought this was to add 4 projects initially.